util/missingmacro.txt

```diff
@@ -128,6 +128,7 @@ DEFINE_OR_DECLARE_STACK_OF_CSTRING(3)
 DEFINE_STACK_OF_STRING(3)
 DEFINE_STACK_OF_CSTRING(3)
 U64(3)
+SSL_OP_BIT(3)
```

Comment:
Don't add to this file! Instead add this to the reserved symbols list in doc/man3/OSSL_CORE_MAKE_FUNC.pod